content/components/segmented-control.mdx

```diff
@@ -0,0 +1,426 @@
+  </Dont>
+</DoDontContainer>
+
+### Contextual information
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="0">
+    <Text as="h4" fontSize={2} fontWeight="bold" m={0}>
+      Label
+    </Text>
+    <Text as="p" mt="0">
+      Segmented controls must have a text label, even if the label is only accessible to screen readers. However, a
+      visible text label is preferred.
+    </Text>
+    <Text as="h4" fontSize={2} fontWeight="bold" m={0}>
+      Caption
+    </Text>
+    <Text as="p" mt="0">
+      An optional caption may be used to provide additional context to help users make a selection or explain how what
+      effect the selection will have. Caption text should be as short as possible.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180623806-23b98c83-e870-4eed-b1f1-635e919a3d2d.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+See the [layout](#layout) section for ways to compose a segmented control with a label.
+
+## Interaction
+
+Selections are mutually exclusive. Once a selection has been made, it can only be deselected by making a new selection.
+
+A segmented control does not require a "Save" button to apply the selection. When a choice is selected, the selection immediately affects an object, state, or view.
+
+## Layout
+
+### Composing the control with a label and caption
+
+#### Single-column
+
+<img
+  src="https://user-images.githubusercontent.com/2313998/180624841-cc4e4bf9-098c-425c-ba84-7df8413c2055.png"
+  role="presentation"
+/>
+
+A single-column layout is the same way form controls (for example, text inputs) are layed out. The single-column layout is useful for preserving horizontal space.
+
+#### Two-column
+
+<Box display="flex" mb={3} flexWrap="wrap" sx={{gap: 3}}>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624842-f3abb168-1d3d-4cdb-a167-2c78059653d5.png"
+    role="presentation"
+    width="456"
+  />
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624843-75a82099-b95f-41f6-b9a9-35adc3e1537e.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+A two-column layout can be used to:
+
+- Align the control with adjacent elements
+- Conserve vertical space
+- Group related controls in a way that is easy to scan
+
+### Fill available width
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Text as="p" mt="0">
+    A segmented control's width can stretch each button evenly to fill the width of the control's parent. This option is
+    intended to make the segmented control easier to adapt into different layouts.
```

Comment:
When should we use full-width vs left-aligned?

Reply:
I had a hard time explaining this because I don't have any specific examples yet. Here are two slightly less vague use-cases:
- A designer might choose to use a full-width segmented control to make a cleaner layout where elements align to the left and right edges
- To make sure each button has an even width. If the text of one button is very long, that button would be very wide and have a heavier visual weight than the other buttons.

I think that extra information would just add confusion, and it's better not to try and specify use cases until we have a better example. What do you think?